lxd/network/openvswitch/ovs.go

```diff
@@ -373,3 +373,137 @@ func (o *OVS) OVNSouthboundDBRemoteAddress() (string, error) {
 
 	return addr, nil
 }
+
+// GetStpPriority returns the STP priority of the OVS bridge.
+// Default STP priority is 32768 (0x8000). This value will be used if the
+// priority cannot be retrieved or if STP is disabled.
+func GetStpPriority(ctx context.Context, bridgeName string) (string, error) {
```

Comment:
Should be GetSTPPriority as acronyms should be uppercased.

https://go.dev/talks/2014/names.slide#5
Initialisms[¶](https://go.dev/wiki/CodeReviewComments#initialisms)